template/apps/values.yaml.jinja

```diff
@@ -35,6 +35,8 @@ ec_services:
   #            the path is always services/<service> within that repo
   #     removed: set to true to remove the service from the cluster
   #     enabled: set to false to stop a service
-  {{ ioc_group }}-ea-test-01:
-    enabled: true
-    targetRevision: {{ services_release }}
\ No newline at end of file
+
+  # example entry
+  #   {{ ioc_group }}-ea-test-01:
+  #     enabled: true
+  #     targetRevision: {{ services_release }}
```

Comment:
My personal feeling is that after I create a repo, I want to deploy it and know it works as is. Therefore I would prefer to not comment this out

Reply:
I have removed test from the default services repo - as per a previous conversation so it won't work if I uncomment that.

Happy to re-consider this.

Reply:
Ah okay I did not realize that this was chosen. Thats fine then